src/pudl/transform/classes.py

```diff
@@ -297,6 +297,47 @@ def enforce_snake_case(
 enforce_snake_case_multicol = multicol_transform_factory(enforce_snake_case)
 
 
+################################################################################
+# Strip Non-Numeric Values
+################################################################################
+class StripNonNumericValues(TransformParams):
+    """Boolean parameter for :func:`strip_non_numeric_values`.
+
+    Stores a named boolean variable that is employed in
+    :func:`strip_non_numeric_values` to determine whether of not the transform
+    treatment should be applied. Pydantic 2.0 will allow validation of these simple
+    variables without needing to define a model.
+    """
+
+    strip_non_numeric_values: bool
+
+
+def strip_non_numeric_values(
+    col: pd.Series, params: StripNonNumericValues | None = None
+) -> pd.Series:
+    """Strip a column of any non-integer values.
```

Comment:
This is a nit but I think this should read, "Strip a column of any non numeric values." I initially read it as just "integer" not "integer type" and values like 3.3 aren't integers in the traditional sense because it's not a whole number. But it is stored as an integer type. I'm assuming you want to keep decimal values in this column.

Reply:
this is a good nit!